open-source-policy/README.md

```diff
@@ -1,23 +1,20 @@
 # CISA Open Source Policy #
 
-This repository contains the official [Open Source Policy](policy.md) of
-[CISA](https://cisa.gov/).
+In general, we work in the open by default.
 
-**[Read CISA's open source policy.](policy.md)**
+**[Read CISA's open source policy](policy.md)** for the official
+[Open Source Policy](policy.md) of [CISA](https://cisa.gov/) and more details
```

Comment:
I think there only needs to be one link to `policy.md` here.

Reply:
Resolved in 37987f9